Automattic/wp-calypso
# Add-ons: Create Storage Add On Card Item

Fixes [1722-gh-Automattic/martech
](1722-gh-Automattic/martech)
### Proposed Changes
Create the card item for the storage add on

### Testing Instructions
1. Checkout this branch
2. Run `yarn` & `yarn start`
3. Go to `http://calypso.localhost:3000/add-ons/YOUR_SITE_SLUG`
4. You should see 2 new cards items - a 50 GB and a 100 GB storage add ons:

![image](https://github.com/Automattic/wp-calypso/assets/20927667/8d59381e-1359-498d-9f94-a042eab4513f)

5. Click on the **Buy add-on** button. You should get redirected to the cart with the correct storage amount applied:

![image](https://github.com/Automattic/wp-calypso/assets/20927667/e8a6798e-732f-4531-90f4-5a73109fd309)
